src/EnergyPlus/HVACUnitarySystem.cc

```diff
@@ -929,33 +1012,69 @@ namespace HVACUnitarySystem {
             if (FirstHVACIteration) {
                 UnitarySystem(UnitarySysNum).iterationCounter = 0;
                 UnitarySystem(UnitarySysNum).iterationMode = 0;
```

Comment:
I have been seeing issues with SP control where the system flow rate was always 0. This is because SP control relies on the TU to set the flow. If someone uses the uncontrolled TU then the unitary system would need to set the flow. So on first HVAC I want to set full flow and then let the TU modulate as needed on subsequent iterations.